src/vue-testing-library.js

```diff
@@ -155,6 +155,9 @@ fireEvent.update = (elem, value) => {
       if (['checkbox', 'radio'].includes(type)) {
         elem.checked = true
         return fireEvent.change(elem)
+      } else if (type === 'file') {
+        Object.defineProperty(elem, 'files', {value})
+        return fireEvent.change(elem)
```

Comment:
```suggestion
      } else if (type === 'file') {
        return fireEvent.change(elem)
```

(also not 100% sure if we should trigger a change event over an input one? 🤔 haven't checked, so it might be okay)

Reply:
Seems pretty okay to me since we know we can't use v-model to an input file and the only reasonable event to use in this case is change.